stock_lot_scrap/models/stock_lot.py

```diff
@@ -102,7 +102,7 @@ def action_scrap_lot(self):
         )
         result["context"] = self.env.context
         if len(scraps) != 1:
-            result["domain"] = "[('id', 'in', %s)]" % scraps.ids
+            result["domain"] = f"[('id', 'in', {scraps.ids})]"
```

Comment:
You don't even need to format it. Odoo can parse the python object directly:
```suggestion
            result["domain"] = [("id", "in", scraps.ids)]